src/test/scala/nodes/images/ImageBenchMarkSuite.scala

```diff
@@ -0,0 +1,143 @@
+package nodes.images
+
+import breeze.linalg._
+import breeze.stats._
+import org.scalatest.FunSuite
+import pipelines.Logging
+import utils._
+import utils.TestUtils._
+
+import scala.util.Random
+
+class ImageBenchMarkSuite extends FunSuite with Logging {
```

Comment:
So, if we're adding this file we also need to change sbt/sbt to increase memory to 4g, or else `sbt/sbt test` will fail if that gets run anywhere